R/package.R

```diff
@@ -19,10 +19,21 @@ as_pkgdown <- function(pkg = ".", override = list()) {
 
   check_string(pkg)
   if (!dir_exists(pkg)) {
+    if (dir.exists(pkg)) { #nolint
```

Comment:
I'd suggest putting a comment here that briefly describes the difference between `~` in base R and fs on windows.